packages/bootstrap/scss/action-sheet/_variables.scss

```diff
@@ -0,0 +1,29 @@
+// Action-sheet
+$action-sheet-width: 372px !default;
+$action-sheet-max-height: 50vh !default;
+$action-sheet-box-shadow: 0 8px 10px -5px rgba(0, 0, 0, .2), 0 16px 24px 2px rgba(0, 0, 0, .14), 0 6px 30px 5px rgba(0, 0, 0, .12) !default;
+
+$action-sheet-border-width: 0 !default;
+$action-sheet-border-radius: 0 !default;
+
+$action-sheet-bg: $component-bg !default;
+$action-sheet-text: $component-text !default;
+$action-sheet-border: $component-border !default;
+
+$action-sheet-item-min-height: 40px !default;
+$action-sheet-item-padding-x: 16px !default;
+$action-sheet-item-padding-y: 8px !default;
+$action-sheet-item-border-width: 1px !default;
+
+$action-sheet-item-icon-font-size: 24px !default;
+$action-sheet-item-icon-color: $primary !default;
+
+$action-sheet-item-title-color: $text-color !default;
+$action-sheet-item-subtitle-color: rgba($action-sheet-item-title-color, .5) !default;
+
+$action-sheet-item--hovered-background: rgb(239, 241, 244) !default;
+$action-sheet-item--focused-background: transparent !default;
+$action-sheet-item--focused-box-shadow: inset 0 0 0 2px rgba(0, 0, 0, .13) !default;
+$action-sheet-item--disabled-opacity: .6 !default;
```

Comment:
Use CSS state names for variables - `hover`, `focus`, etc. (Yes, we have "hovered" variables - they are legacy).

Also, why do you use a double dash syntax for these var? I believe singe dash is the way to go.